contributor/03-contributor-ethos.md

```diff
@@ -0,0 +1,64 @@
+# Draft comments
+
+ Outline for this section
+
+     Contributors by definition work in teams that they are not familiar with. This section explains how to avoid common pitfalls when crossing team boundaries.
+
+ ## Article
+
+In the last segment we have motivated why you would want to reuse components and become active as a Contributor. This artice is sharing best practices on how to successfully contribute your changes to the host team's code base.
+
+ ### Contributor Ethos
+
+An InnerSource Contributor trying to make a contribution to the host team essentially is a guest in their home. In general a good guest is expected to behave in a certain way:
+
+* They knock at the door.
+* They anticipate and follow house rules.
+* They understand they are not the home owner and acty accordingly.
+
+How do these expectations apply to InnerSource projects?
+
+#### Getting in
+
+In the real world their are several ways that guests are invited in:
+
+* An "open" sign lets them know they can walk right in without knocking.
+* Knock on the door unexpected.
+* Knock on the door only when invited.
+
+With InnerSource projects there are several ways that code gets in:
+
+Sometimes you just open a pull request. At times you may need to get a feature request accepted before opening the pull request that fulfills it.
+
+In some organisations you may need to talk to the host team in person in addition to interacting with the project digitally. 
```

Comment:
Suggest putting this point after the following paragraph so that the ways of getting in are ordered from lowest-touch (just open a PR) to highest touch (meet the host team first).

Reply:
I like that approach. 
I do think that we should add technically mediated "in-person" conversation though, i.e. a voice/video call or the likes. Sometimes even a chat will work wonders. 
"Digital interaction" could be a clearer - mail, voice, video, issue - all of those are digital. Even an appointment to discuss f2f has digital aspects. How about using the division of synchronous vs. asynchronous conversation here? Initiating a synchronous conversation (kitchenette+coffee, phonebooth, voice, video) with someone often helped for that purpose in my observation.
Adding a review here.

```suggestion
In some organisations you may need to talk to the host team in a synchronous manner, e.g. around a coffee, by means of a video of voice chat, in addition to interacting with the project asynchronously, e.g. by means of issue trackers or email to establish some trust. 
```

Reply:
You mean something along what I talked about there maybe:

https://drive.google.com/drive/u/0/folders/0ByhNYwghjRyeOVlMN0lBYndsdXc (starting page 47 and following)?

Reply:
You've done a lot of work!  For me that link goes to a folder rather than a specific document, though.